docs/source/en/model_doc/videomae.mdx

```diff
@@ -25,6 +25,7 @@ Tips:
 
 - One can use [`VideoMAEFeatureExtractor`] to prepare videos for the model. It will resize + normalize all frames of a video for you.
 - [`VideoMAEForPreTraining`] includes the decoder on top for self-supervised pre-training.
+- You can check out [this notebook](https://github.com/huggingface/notebooks/blob/main/examples/video_classification.ipynb) that shows how to fine-tune a VideoMAE model on a custom dataset.
```

Comment:
We now include a separate "Resources" section in the docs of models, see BERT as an example. Would be great to also add that

Reply:
Done. @stevhliu could you take a look?